### DISCUSSION TITLE
Next Github Cmapus Experts application starting date

### DISCUSSION THREAD

**Discussion by Ranojitdas** [community/community#139301] [GitHub Education]
### Hi everyone,

when will the application form open for Github Campus experts ?

**Comment by davevad93** [comment]
Hi @Ranojitdas ,

In the [docs](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/use-github-at-your-educational-institution/applying-to-be-a-github-campus-expert) it says that "***applications to the program open in February and August, and you’ll have a full month to apply.***", so the next time would be February 2025.

**Comment by Injamul2k24** [comment]
Hi Ranojitdas,
 Yes it start again in Feb 2025
